vlib/v/fmt/tests/global_keep.vv

```diff
@@ -6,5 +10,8 @@ __global (
 	foo  = int(5)
 )
 __global (
-	a int
+	a        int
```

Comment:
This change is caused by `v fmt` due to the latter declaration. I think this is a bug.

Reply:
it is not